cime_config/machines/config_machines.xml

```diff
@@ -595,6 +595,9 @@
     <SUPPORTED_BY>e3sm</SUPPORTED_BY>
     <MAX_TASKS_PER_NODE>64</MAX_TASKS_PER_NODE>
     <MAX_MPITASKS_PER_NODE>64</MAX_MPITASKS_PER_NODE>
+    <MAX_MPITASKS_PER_NODE compiler="gnugpu">16</MAX_MPITASKS_PER_NODE>
```

Comment:
There are 8 GCDs on crusher, why is 16 used below?

Reply:
IIRC, that's what was best in MMF runs.